aporia/manifest.json

```diff
@@ -0,0 +1,65 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "1b4bbfb0-659e-4c67-b2a8-1d11853701b1",
+  "app_id": "aporia",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Cloud-Native ML Monitoring and Explainability",
+    "title": "Aporia",
+    "media": [
+      {
+        "caption": "Aporia Dashboard in DataDog",

```

Comment:
Please ensure all images have a 16:9 aspect ratio or else they will get cut off in the media carousel